phoenix-core/src/test/java/org/apache/phoenix/jdbc/PhoenixTestDriver.java

```diff
@@ -92,7 +93,8 @@ public synchronized Connection connect(String url, Properties info) throws SQLEx
     public synchronized ConnectionQueryServices getConnectionQueryServices(String url, Properties infoIn) throws SQLException {
         checkClosed();
         final Properties info = PropertiesUtil.deepCopy(infoIn);
-        ConnectionInfo connInfo = ConnectionInfo.create(url, null, info);
+        boolean isServerConnection = Boolean.valueOf(info.getProperty(QueryUtil.IS_SERVER_CONNECTION));
+        ConnectionInfo connInfo = ConnectionInfo.create(url, null, info, isServerConnection);
```

Comment:
Why do we want to pass as a separate argument for `isServerConnection`?  It is present in `info` object anyways.

Reply:
Pushed the refactor, no changes in Drivers needed.